tox.ini

```diff
@@ -22,6 +22,14 @@ deps =
     zope.proxy
     zope.security
 
+[testenv:pypy3]
+# On certain platforms, at least OS X, pypy3 does not correctly read
+# a LANG environment variable; when cpython3.2 would return 'utf-8'
+# from locale.getpreferredencoding(), pypy3 returns 'us-ascii', causing
+# two test failures. Fix that by setting something it does read.
```

Comment:
It's true that locally I'm on tox 2.0.1. That's the only version I've used during this testing. My comment may be incorrect, though this did fix the issue.

In debugging this problem, I added `print` lines to the stdlib's `cgi` module in its `FieldStorage` class. Under tox 2.0.1, CPython3.2 said that `locale.getpreferredencoding()` returned `UTF-8`, while PyPy3 returned `US-ASCII`. The `cgi` modules are identical in the two implementations. This fixed the problem. 

I'm open to any suggestions. I can upgrade `tox`, but I've seen a similar issue under `buildout` with PyPy3 (e.g., installing the `zopetoolkit` buildout failed because `collective.recipe.cmd` opened a file in the default encoding; even though my `LANG` is set to `en_US.UTF-8`, PyPy3 failed to read the file), suggesting to me the issue is in PyPy3 itself.